cmd/pulumi.go

```diff
@@ -175,6 +175,7 @@ func NewPulumiCmd() *cobra.Command {
 	cmd.AddCommand(newPluginCmd())
 	cmd.AddCommand(newVersionCmd())
 	cmd.AddCommand(newHistoryCmd())
+	cmd.AddCommand(newViewTraceCmd())
```

Comment:
Happy to hide this under the debug flag if that seems desirable.

Reply:
I wonder if there is a grouping of commands like this that are "diagnostic" commands so that we could at least move this out of the top-level?